bucket_AB/ART/specification

```diff
@@ -0,0 +1,43 @@
+DEF[PORTVERSION]=	1.24.4
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		ART
+VERSION=		${PORTVERSION}
+KEYWORDS=		graphics
+VARIANTS=		std
+SDESC[std]=		Raw image processing program
+HOMEPAGE=		https://art.pixls.us/
+CONTACT=		Michael_Neumann[mneumann@ntecs.de]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		GITHUB/artpixls:ART:${PORTVERSION}
+DISTFILE[1]=		generated:main
+
+SPKGS[std]=		primary
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+USES=			cmake c++:primary jpeg tiff pkgconfig
+
+# TODO: OpenColorIO
+CMAKE_ARGS=		-DENABLE_OCIO:BOOL=OFF
+
+BUILD_DEPENDS=		libcanberra:dev:std
+			fftw:single:float
+			libraw:dev:std
+			exiv2:dev:std
+
+BUILDRUN_DEPENDS=	exiv2:primary:std
+			lcms2:dev:std
+			libcanberra:primary:std
+			lensfun:primary:std
+
+RUN_DEPENDS=		libcanberra:gtk3:std
+
+GNOME_COMPONENTS=	gtk3 gtkmm30 librsvg glibmm24
+
+LICENSE=		GPLv3+:primary
```

Comment:
this is not technically wrong but conventionally the LICENSE block follows the OPTIONS block